app/models/user.rb

```diff
@@ -1,2 +1,4 @@
 class User < ApplicationRecord
+  has_many :comments
```

Comment:
- [ ] as part of the requirements the User model should have a method that returns 3 most recent posts , kindly add that :pray: